chatsky/responses/rag.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, Any
+from sentence_transformers import SentenceTransformer
+from qdrant_client import QdrantClient
+from chatsky.responses.base_response import BaseResponse
+
+
+class RAGPromptBuilder(BaseResponse):
+    def __init__(self, vector_store_url: str, retriever_model: str, **kwargs):
+        super().__init__(**kwargs)
+        self.vector_store = QdrantClient(url=vector_store_url)
+        self.retriever = SentenceTransformer(retriever_model)
+
+    def execute(self, user_query: str, ctx: Dict[str, Any], **kwargs) -> Dict[str, Any]:
+        if "cached_rag_context" in ctx:
+            context = ctx["cached_rag_context"]
```

Comment:
`context` and `ctx` are similar parameter names which can be confusing. It's better to rename `context` to something else (e.g. `rag_context`).